pkg/standalone/run.go

```diff
@@ -369,3 +343,13 @@ func Run(config *RunConfig) (*RunOutput, error) {
 		DaprGRPCPort: daprGRPCPort,
 	}, nil
 }
+
+func getComponentsPath(config *RunConfig) (string, error) {
+	if config.ComponentsPath == "" {
+		componentsPath := getDefaultComponentsFolder()
+		return componentsPath, nil
+	}
+
+	_, err := os.Stat(config.ComponentsPath)
```

Comment:
This should check if the path exists as well.  The usage could clarify if the cli expects the specified path to exist ahead of time or if the cli will create it.

Reply:
Not sure if I understood you. Updated both code paths to test for dir to exist and added more doc in the README to clarify. Also, the idea behind the default components folder is that the new users might not even know what the components folder is for and we create it for them to help them get started.

Reply:
> Also, the idea behind the default components folder is that the new users might not even know what the components folder is for and we create it for them to help them get started.

Yes I mean in the case where the user specified --components-path.

Reply:
Added